src/map/entities/npcentity.h

```diff
@@ -33,7 +33,9 @@ class CNpcEntity : public CBaseEntity
     uint32       m_flags;
     uint8        name_prefix;
     uint8        widescan;
-    bool         m_triggerable = false;
+    bool         m_triggerable  = false;
+    bool         m_alwaysRender = false;
+    void         setAlwaysRender(bool alwaysRender); // Used for permanent features in a zone such as decorations / fireworks
```

Comment:
You've made a setter function here, but you change the field directly in [lua_baseentity.cpp](https://github.com/AirSkyBoat/AirSkyBoat/pull/3249/files#diff-dcaa3294e15a4adab745513619c6bee1a3283d70270f4f6b0bd546f161bbd895)